### ISSUE TITLE
Ability to add new custom fields and data

### ISSUE THREAD

**Issue by akhan4u** [jeffrey-hui/workpls#6]
Hi there,

I found the extensions super useful, but however It would be great if you can add functionality to add custom fields and values. This helps with adding all relevant fields that any job portal may have and help accordingly.

**Comment by jeffrey-hui** [comment]
I agree that this would be super useful. Unfortunately, it is programmatically difficult due to the variability of each individual form because of how lever/greenhouse clients can choose to add custom fields with any element tag they want.
